src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,63 @@ title: Sign up for CodeStream
+If the email address you're using with your code host isn't your work email, you
+should sign up with your work email address instead.
 
-If you sign up via CodeStream, your next step will be to confirm your email address by entering a code sent to you via email. You can paste the code into any of the boxes rather than typing each number individually.
+If you sign up using your email address, your next step will be to confirm the
+email address by entering a code sent to you via email. You can paste the code
+into any of the boxes rather than typing each number individually.
 
 ![Confirm Email](./images/EmailConfirmation1.png)
 
-## CodeStream organizations [#organizations]
-
-A CodeStream organization is where you and your teammates will discuss code. Similar to a Slack workspace, all of the developers in your company are in the same CodeStream organization. CodeStream's activity feed keeps things relevant for you by showing activity related to the code you have open in your IDE. 
-
-The discussions about code build up a knowledge base that is a company-wide resource, so the only reason to have multiple organizations on CodeStream is if you truly need separation. For example, you might have an organization for your day job and another for an open-source project you work on. Or maybe you're a consultant that's a member of different CodeStream organizations for each of your clients.
-
 ## Create or join an organization [#create-org]
 
-If you're invited to join an organization on CodeStream, sign up with the same email address the invitation was sent to. You'll automatically be added to that organization.
+A CodeStream organization is where you and your teammates will discuss code.
+Similar to a Slack workspace, all of the developers in your company can be in
+the same CodeStream organization. 
 
-If you're the first person from your company to sign up for CodeStream you can go ahead and create a new organization. Otherwise, there may be existing CodeStream organizations available for you to join based on your email domain. If you think your company is already on CodeStream, but don't see an organization to join, make sure that you've signed up with your work email.
+If you're invited to join an organization on CodeStream, sign up with the same
+email address the invitation was sent to and you'll automatically be added to
+that organization. Alternatively, there may be existing CodeStream organizations
+available for you to join based on your email domain.
 
 ![Create or join an organization](./images/CreateOrJoinOrg.png)
 
-If you decide to create an organization you'll be asked to give it a name and, if you signed up with your work email address, anyone else with that email domain can join the organization.
-
-![Create an organization](./images/CreateOrg.png)
+If you weren't invited, and there are no existing organizations for you to join,
+a new organization will be created for you.
```

Comment:
```suggestion
If you're the first one from your email domain to create a CodeStream account, a new organization is created. Every new user from your domain that follows is added to this organization.
```

Reply:
Per my previous comment, this isn't accurate. Things aren't automatically tied to an email domain. Like Slack, it's a feature that needs to be turned on by an admin. Even with it on, there's nothing stopping 10 people from acme.com from creating their own orgs if that's what they want.

Reply:
I think we need to add what you just wrote in this comment to this doc. I don't think it's at all clear as it's presently written. Hence my continual confusion about how this actually works, I think :D